node/full_client_test.go

```diff
@@ -56,14 +56,15 @@ func getRandomValidatorSet() *tmtypes.ValidatorSet {
 	}
 }
 
+var genesisValidatorKey = ed25519.GenPrivKey()
+
 // TODO: use n and return n validators
 func getGenesisValidatorSetWithSigner(n int) ([]tmtypes.GenesisValidator, crypto.PrivKey) {
-	validatorKey := ed25519.GenPrivKey()
```

Comment:
why does this not work and simply using the global variable works?

Reply:
This was creating a new validator key everytime this function was being called, so the nodes had different validator sets. The aggregator hash was being set from the first node (in this test case first node is aggregator) and the rest of the nodes were failing the hash check.

Reply:
got it. may be you can remove the function `getGenesisValidatorSetWithSigner` and have GEN_VAL_SET and GEN_KEY as constants?

Reply:
Created #860